src/lib.rs

```diff
@@ -42,32 +45,13 @@ struct CustomVertex {
     uv: [f32; 2],
 }
 
-type Result<T> = core::result::Result<T, HRESULT>;
```

Comment:
Let's reuse https://microsoft.github.io/windows-docs-rs/doc/windows/core/type.Result.html here (and also instead of `Result<..., ()>`) so we can `?` the errors instead of unwrapping/`match`ing all of them

Reply:
I actually googled a bit to see how I could do this, thanks for showing me this. Cleans up the code a lot!